assets/scss/app.scss

```diff
@@ -22,13 +22,15 @@ body {
   @include core-19;
 }
 
-//h1 {
-//  @include bold-24;
-//  margin: em(15) 0 em(12);
-//  .query {
-//    font-weight: normal;
-//  }
-//}
+.heading-large, h1 {
+  font-size: 19px;
+  margin-top: 10px;
+
+  @include media(desktop) {
+    font-size: 32px;
+    margin-top: 20px;
```

Comment:
The media query is just to give the design a "responsive" nature. There doesn't seem to be anything in the HMPO toolkit that would serve in place of these styles, so might just have to bite the bullet for now...